client/packages/invoices/src/Prescriptions/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -8,57 +8,56 @@ import {
   useTranslation,
   useFormatCurrency,
 } from '@openmsupply-client/common';
-import { useInsurance, usePrescription } from '../../api';
-import { usePrescriptionGraphQL } from '../../api/usePrescriptionGraphQL';
+import { usePrescription } from '../../api';
 
 export const PricingSectionComponent = () => {
   const t = useTranslation();
   const c = useFormatCurrency();
 
   const {
-    query: { data: prescriptionData}
+    query: { data: prescriptionData },
   } = usePrescription();
   const pricing = prescriptionData?.pricing;
   if (!pricing) return null;
 
-  const insuranceId = prescriptionData.nameInsuranceJoinId;
-  const { storeId } = usePrescriptionGraphQL()
-  const {
-    query: {data: insuranceData}
-   } = useInsurance(insuranceId, storeId);
-
   return (
     <DetailPanelSection title={t('heading.pricing')}>
       <Grid container gap={0.5}>
-        
-        {insuranceData && (
-        <>
-        <PanelRow>
-          <PanelLabel>{t('label.insurance-provider-name')}</PanelLabel>
-          <PanelField>
-            {insuranceData.insuranceProviders?.providerName}
-          </PanelField>
-        </PanelRow>
+        {prescriptionData.insurancePolicy && (
+          <>
+            <PanelRow>
+              <PanelLabel>{t('label.insurance-provider-name')}</PanelLabel>
+              <PanelField>
+                {
+                  prescriptionData.insurancePolicy.insuranceProviders
+                    ?.providerName
+                }
+              </PanelField>
+            </PanelRow>
 
-        <PanelRow>
-          <PanelLabel>{t('label.insurance-policy-number')}</PanelLabel>
-          <PanelField>
-            {insuranceData.policyNumber}
-          </PanelField>
-        </PanelRow>
+            <PanelRow>
+              <PanelLabel>{t('label.insurance-policy-number')}</PanelLabel>
+              <PanelField>
+                {prescriptionData.insurancePolicy.policyNumber}
+              </PanelField>
+            </PanelRow>
 
-        <PanelRow>
-          <PanelLabel>{t('label.insurance-discount-amount')}</PanelLabel>
-          <PanelField>
-            {c(prescriptionData.insuranceDiscountAmount ?? 0)}
-          </PanelField>
-        </PanelRow>
+            <PanelRow>
+              <PanelLabel>{t('label.insurance-discount-amount')}</PanelLabel>
+              <PanelField>
+                {c(prescriptionData.insuranceDiscountAmount ?? 0)}
+              </PanelField>
+            </PanelRow>
 
-        <PanelRow>
-          <PanelLabel>{t('label.insurance-discount-percentage')}</PanelLabel>
-          <PanelField>{insuranceData.discountPercentage}% </PanelField>
-        </PanelRow>
-        </>
+            <PanelRow>
+              <PanelLabel>
+                {t('label.insurance-discount-percentage')}
+              </PanelLabel>
+              <PanelField>
+                {prescriptionData.insuranceDiscountPercentage ?? 0}%{' '}
```

Comment:
This is actually a bug fix, using the prescription data which should have been used before